src/steps/ms-defender/machine/__snapshots__/index.test.ts.snap

```diff
@@ -85,13 +85,17 @@ Object {
       "computerDnsName": "testmachine3",
       "createdOn": undefined,
       "defenderAvStatus": "NotSupported",
-      "deviceId": "unknown",
```

Comment:
Setting this value to an arbitrary string is not a good idea. It's far better to just have it be empty if we don't know it.